site/style.css

```diff
@@ -67,7 +67,7 @@ h3 {
 }
 
 h1 span {
-	color: #AAAAAA;
+	color: #aaaaaa;
 }
 
 fieldeset td, fieldeset th {
```

Comment:
Yeah, exactly my point :P

Nevertheless, I addressed your previous comments in this branch, but for any others just make a PR.